.pre-commit-config.yaml

```diff
@@ -40,4 +40,4 @@ repos:
     # F401 = F811; This happens for fixtures for indirect parametrize
     args:
     - --max-line-length=88
```

Comment:
Are you sure you want to ignore line-lengths?
If yes, I guess we don't need a max length

Reply:
flake8 uses ``--max-line-length=88`` and E501 for different things. I don't want the tooling to choke on flake8 complaining about a url in documentation string being too long. Instead, we can rely on **black** for fixing line-length / format in a saner maner, and then rely on flake8 to take care of the language integrity things that **black** does not cater to.